backend/pipelined/handle.go

```diff
@@ -19,6 +19,11 @@ const (
 	DefaultSocketTimeout uint32 = 60
 )
 
+type handlerExtensionUnion struct {
```

Comment:
In the C/C++ universe, the term 'union' means 'two separate types that share the same memory space', I assume it means something different for go? I'm still in learning mode here.

Reply:
Go doesn't have unions, but you can simulate them through anonymous embedding.